sdcv.el

```
@@ -596,7 +608,7 @@ Argument DICTIONARY-LIST the word that need transform."
     (dolist (word words)
```

Change `dolist` into `cl-dolist`. Otherwise, `cl-return` does not have a way to exit. This loop is relying on implicit cl-block.

---

You are absolutely right.  But after I look throught the script, it seems only this `dolist` actually uses cl-lib, maybe it should be
```diff
-           (return word)
+           word
```
and rid of the cl-lib dependency all together ?

---

you can use throw/catch instead of cl-return:
```elisp
(catch 'result
  (dolist (word words)
    (if ... (throw 'result word) ...)))
```

---

> you can use throw/catch instead of cl-return:
> 
> ```emacs-lisp
> (catch 'result
>   (dolist (word words)
>     (if ... (throw 'result word) ...)))
> ```

Done #24 , I think this PR can be close now, but free to comment or re-open if necessary.